examples/gaussian_process/plot_gpr_dim_selection.py

```diff
@@ -0,0 +1,110 @@
+"""
+=========================================================
+Illustration of SelectDimensionKernel
+=========================================================
+A simple two-dimensional regression example computed in two different ways:
+1. With a product of two RBF kernels on each feature.
+2. With one anisotropic RBF kernels applied on both feature.
+
+The figures illustrate the property of SelectDimensionKernel when applied on
+different features of input data.
+"""
```

Comment:
Could you briefly describe in the docstring what we see in the figure (what is the true function, what are the GP's  prediction, what is the training data etc.)?